openpathsampling/pathmover.py

```diff
@@ -601,7 +606,14 @@ def move(self, sample_set):
             )
 
         # 4. accept/reject
-        accepted, details = self._accept(trials)
+        accepted, acceptance_details = self._accept(trials)
+
+        # update details
+        kwargs = {}
+        kwargs.update(call_details)
+        kwargs.update(acceptance_details)
```

Comment:
Side note: 2.0 will treat the acceptance as a separate mover, meaning that it will remove the risk of collision in this update.

(Until then, contributors just need to know that `metropolis_acceptance` and `metropolis_random` are not allowed as names of details in their movers -- I don't think it'll be a problem).